virttest/qemu_vm.py

```diff
@@ -1637,7 +1638,8 @@ def sort_key(dev):
             usbs = ("oldusb",)  # Old qemu, add only one controller '-usb'
         for usb_name in usbs:
             usb_params = params.object_params(usb_name)
-            for dev in devices.usbc_by_params(usb_name, usb_params):
+            parent_bus = _get_pci_bus(devices, usb_params, "usbc", True)
```

Comment:
It seems usb devices will still attach to `{'aobject': params.get('pci_bus', 'pci.0')}`.
Do you plan to enable attach usb devices to a specified parent bus by using `params["usbc_pci_bus"]`? What are the values for this param?
Otherwise, there is no need to change this part.

Reply:
The `{'aobject': params.get('pci_bus', 'pci.0')}` allows one to plug the usb controller to the bus they want to, which is used by the `pci_devices` test to insert controllers into switches or busses by using something like `pci_bus_test_device1 = test_bridge1`.

It used to work that way before, but then the `_get_pci_bus` was introduced some devices were addressed, but usb controllers were not, which broke the `pci_devices` test.

Reply:
OK, Thanks, the doc string of `_get_pci_bus` may need update as well.